tests/common.h

```diff
@@ -251,7 +251,7 @@ namespace witest
         bool doesThrow = false;
         bool doesCrash = DoesCodeCrash([&]()
         {
-            doesThrow = DoesCodeThrow(callOp);
+            doesThrow = DoesCodeThrow(wistd::forward<Lambda>(callOp));

```

Comment:
Was this actually a warning? Forwarding lambdas that aren't copied/moved - while not necessarily a bad thing - is technically unnecessary.

Reply:
I'll try removing. There was a complaint about "unused unnamed temporary objects" that prompted this, but it's possible I was holding the report wrong.

Reply:
Could it possibly be related to the `(void)` you added to `DoesCodeThrow`? Before the change, this function will capture `callOp` as either an l-value or r-value reference and pass as an l-value reference to `DoesCodeThrow`. There shouldn't be any "unused unnamed temporary objects" created